js/accordionView.js

```diff
@@ -1,86 +1,86 @@
+      this.listenTo(this.model.get('_children'), {
+        'change:_isActive': this.onItemsActiveChange,
+        'change:_isVisited': this.onItemsVisitedChange
+      });
+    },
 
-        postRender: function() {
-            this.setReadyStatus();
+    postRender: function() {
+      this.setReadyStatus();
 
-            if (this.model.get('_setCompletionOn') === 'inview') {
-                this.setupInviewCompletion();
-            }
-        },
+      if (this.model.get('_setCompletionOn') === 'inview') {
+        this.setupInviewCompletion();
+      }
+    },
 
-        checkIfResetOnRevisit: function() {
-            var isResetOnRevisit = this.model.get('_isResetOnRevisit');
+    checkIfResetOnRevisit: function() {
+      var isResetOnRevisit = this.model.get('_isResetOnRevisit');
 
-            // If reset is enabled set defaults
-            if (isResetOnRevisit) {
-                this.model.reset(isResetOnRevisit);
-            }
-        },
+      // If reset is enabled set defaults
+      if (isResetOnRevisit) {
+        this.model.reset(isResetOnRevisit);
+      }
+    },
 
-        onClick: function(event) {
-            event.preventDefault();
+    onClick: function(event) {
+      event.preventDefault();
 
-            this.model.toggleItemsState($(event.currentTarget).parent().data('index'));
-        },
+      this.model.toggleItemsState($(event.currentTarget).parent().data('index'));
+    },
 
-        onItemsActiveChange: function(item, isActive) {
-            this.toggleItem(item, isActive);
-        },
+    onItemsActiveChange: function(item, isActive) {
+      this.toggleItem(item, isActive);
+    },
 
-        onItemsVisitedChange: function(item, isVisited) {
-            if (!isVisited) return;
+    onItemsVisitedChange: function(item, isVisited) {
+      if (!isVisited) return;
 
-            var $item = this.getItemElement(item);
+      var $item = this.getItemElement(item);
 
-            $item.children('.accordion-item-title').addClass('visited');
-        },
+      $item.children('.accordion__item-btn').addClass('is-visited');
+    },
 
-        toggleItem: function(item, shouldExpand) {
-            var $item = this.getItemElement(item);
-            var $body = $item.children('.accordion-item-body').stop(true, true);
+    toggleItem: function(item, shouldExpand) {
+      var $item = this.getItemElement(item);
+      var $body = $item.children('.accordion__item-content').stop(true, true);
 
-            $item.children('.accordion-item-title')
-                .toggleClass('selected', shouldExpand)
-                .attr('aria-expanded', shouldExpand);
-            $item.find('.accordion-item-title-icon')
-                .toggleClass('icon-plus', !shouldExpand)
-                .toggleClass('icon-minus', shouldExpand);
+      $item.children('.accordion__item-btn')
+          .toggleClass('is-selected', shouldExpand)
+          .attr('aria-expanded', shouldExpand);
+      $item.children('.accordion__item-btn')
```

Comment:
These should be chained rather than making the same lookup, i.e.
```js
$item.children('.accordion__item-btn')
    .toggleClass('is-selected is-open', shouldExpand)
    .toggleClass('is-closed', !shouldExpand)
    .attr('aria-expanded', shouldExpand);
```